GeometryService/inc/GeometryService.hh

```diff
@@ -69,7 +95,7 @@ public:
 
     }
 
-    bool isStandardMu2eDetector() const { return standardMu2eDetector_; }
+    bool isStandardMu2eDetector() const { return _standardMu2eDetector; }
```

Comment:
Hi Rob,   technically (some?) names starting with an underscore are reserved for C++ implementers, so it is a bad practice to introduce them into user code.    Trailing underscores should be preferred.
Andrei

Reply:
While what you say is true, the majority of the Mu2e code base has prepended underscores.  Changing pre to post underscores was not part of the code consistency discussion.  A mix of pre and post-pended underscores is clearly an inconsistency, which was agreed by all as something to avoid.   Rob's commit made the file more self-consistent with the minimum of change, so satisfies these goals.

Reply:
These days I use trailing on new code.  For old code I value consistency within the file over the small risk.